__tests__/app.test.js

```diff
@@ -276,5 +276,87 @@ describe('GET /api/', () => {
             })
         })
     });
+
+    describe.only('POST /api/articles/:article_id/comments', () => {
+        test('200: posts a new comment and responds with the new comment object ', () => {
+            const body = {username: "icellusedkars", body: 'nice article'}
+
+            return request(app)
+            .post('/api/articles/3/comments')
+            .send(body)
+            .expect(200)
+            .then(({ body : {new_comment} }) => {
+                expect(new_comment).toEqual(
+                    expect.objectContaining({
+                        comment_id: expect.any(Number),
```

Comment:
I think we can explicitly say that we expect this to be 3.

Reply:
Will do thanks Rose!